README.md

```diff
@@ -362,3 +373,5 @@ The name **LeakCanary** is a reference to the expression [canary in a coal mine]
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
+[snap]: https://oss.sonatype.org/content/repositories/snapshots/
```

Comment:
we've used in place markdown links everywhere else, no reason this should be different.